pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ast/StmtLoop.java

```diff
@@ -30,31 +30,31 @@
 
 package com.cubrid.plcsql.compiler.ast;
 
-import org.antlr.v4.runtime.ParserRuleContext;
-import java.util.List;
+import com.cubrid.plcsql.compiler.ast.loopOpt.SqlUse;
 import java.util.LinkedList;
-import java.sql.PreparedStatement;
+import java.util.List;
+import org.antlr.v4.runtime.ParserRuleContext;
 
 public abstract class StmtLoop extends Stmt {
 
-    public final LoopOptimizable loopOptimizable;
+    public final LoopOptimizables loopOptimizables;
```

Comment:
```suggestion
    private final LoopOptimizables loopOptimizables;
```
관리의 용이성을 위하여 public 보다는 private 선언하는 것이 좋아보입니다.

Reply:
getter 호출 없이 접근하고 싶어서 (성능면) 그냥 두려고 합니다.